docs/authorization-configuration.md

```diff
@@ -136,26 +136,21 @@ auth: {
 - **default**: `60` seconds
 #### thirdPartyCookiesScriptLocation
 - **type**: string
-- **description**: is the URL to the page containing third-party cookies support check. For details, see [Third-party cookies and silent token refresh section](#Third-party-cookies-and-silent-token-refresh).
+- **description**: is the URL to the page containing third-party cookies support check. For details, see [Silent token refresh section](#silent-token-refresh).
 #### userInfoFn
 - **type**: function
 - **description**: provides a function to get user information. It returns a promise of a **userinfo** object which can contain **name**, **email** and **picture** (value is a URL to the image). **Name** or **email** are displayed in the profile drop-down menu and the user’s profile picture is displayed in the top navigation.
 #### profileStorageInterceptorFn
 - **type**: function
 - **description**: provides a function to mutate the profile values of the [JSON Web Token (JWT)](https://jwt.io) before it gets stored in Luigi. It allows the removal of values like **email** to comply with data privacy restrictions. Since it is async, you could use it to enrich the profile data, but it should not get mixed up with **userInfoFn** which is specifically designed to define user information.
 
-### Third-party cookies and silent token refresh
+### Silent token refresh
 
 The OpenID Connect configuration allows you to specify the **automaticSilentRenew** option. When set to `true`, Luigi attempts to automatically renew the token in the background before it expires. Be aware that this mechanism requires the browser to support [third-party cookies](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Third-party_cookies).
 
-To detect whether the user's browser supports the mechanism, use the script in the [`third-party-cookies`](https://github.com/SAP/luigi/tree/main/core/third-party-cookies) catalog. Deploy this file on a domain different from your main application's and set **thirdPartyCookiesScriptLocation** to the `init.html` file. During initialization, Luigi detects the cookies support and produces a warning in the console if cookies are disabled in the user's browser.
+When Luigi fails to renew the token and then logs the user out, it adds the `?reason=tokenExpired` query parameter to the logout page redirect URL. Use this parameter to set a logout page.
 
-When Luigi fails to renew the token and then logs the user out, it adds the `?reason=tokenExpired&thirdPartyCookies=[VALUE]` query parameters to the logout page redirect URL. Luigi replaces **[VALUE]**  with one of these options:
-- `disabled` means that third party cookies are disabled.
-- `enabled` means that the browser supports third party cookies.
-- `not_checked` means that the script was not provided in **thirdPartyCookiesScriptLocation** or it could not be loaded.
-
-Use these parameters to set a logout page.
+Notice - starting from version 3.0 Luigi doesn't support third-party cookies in any way.
```

Comment:
I like this Notice. But I feel in this way it would be overlooked. We usually have some custom warning box for such cases. Check screenshot below and respective docu part to do sth similar 

<img width="923" alt="Screenshot 2024-05-29 at 17 12 31" src="https://github.com/SAP/luigi/assets/18335489/8db4e20b-dd4e-4bab-96d2-824a009c9405">